pkg/api/tag_test.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
```

Comment:
It would be good to have a test where multiple chunks are uploaded with the same tag uid.

Reply:
There is a test "get-tag-indirectly-and-use-it-to-upload-chunk" .. but i will add a direct scenario also

Reply:
We should check for other tag counters, like Total for multiple chunks and that can be most clearly be done in a specific test for it.

Reply:
its done now

Reply:
thanks @jmozah